docs/installation.rst

```diff
@@ -1,11 +1,6 @@
-:orphan:
```

Comment:
Please don't merge yet. I realized this will break the top nav bar: 

<img width="237" alt="Screenshot 2023-08-21 at 7 50 02 AM" src="https://github.com/Qiskit/qiskit/assets/14852634/4ee8a29d-1895-4b02-80a1-e781921a4173">

We definitely want to go ahead with renaming the page title to Installation, but the HTML page needs to still be `getting_started.html` for now. You can do that by restoring the file name, but still changing the page title like you do here.

I think the only way for a graceful migration here would be to set up a redirect. But given that the docs will change soon to a new structure, I'm not sure it matters enough to rename this page. I imagine we have several third-party links to this page, so probably simplest to keep it this name for now.

Reply:
Great spot – should be fixed in [7f85048](https://github.com/Qiskit/qiskit/pull/10328/commits/7f85048b30858dcfe2881cf2049bc865c13e012b)